test/blob_test.js

```diff
@@ -1,4 +1,4 @@
-suite.only("Azure Blob", function() {
+suite("Azure Blob", function() {
```

Comment:
Relying on sleep() calls in tests concerns me a little.  Is there a reason we could do something like `new Date(now().getTime() - 1000)`?